atlasdb-timelock-server/src/main/java/com/palantir/atlasdb/timelock/TimeLockServerLauncher.java

```diff
@@ -58,19 +58,35 @@ private static void registerResources(
             TimeLockServer serverImpl) {
         Map<String, TimeLockServices> clientToServices = createTimeLockServicesForClients(
                 serverImpl,
-                configuration.clients());
+                configuration.clients(),
+                environment);
 
         environment.jersey().register(HttpRemotingJerseyFeature.DEFAULT);
         environment.jersey().register(new TimeLockResource(clientToServices));
     }
 
     private static Map<String, TimeLockServices> createTimeLockServicesForClients(
             TimeLockServer serverImpl,
-            Set<String> clients) {
+            Set<String> clients,
+            Environment environment) {
         ImmutableMap.Builder<String, TimeLockServices> clientToServices = ImmutableMap.builder();
         for (String client : clients) {
-            clientToServices.put(client, serverImpl.createInvalidatingTimeLockServices(client));
+            TimeLockServices invalidatingTimeLockServices = serverImpl.createInvalidatingTimeLockServices(client);
```

Comment:
Consider extracting a method and turning the for loop into `clients.forEach(...)`.